mappings/maestro_file_centric.json

```diff
@@ -0,0 +1,189 @@
+{
+  "aliases": null,
+  "settings": {
+    "index.stuff": null
```

Comment:
reminder to fill the number of shards

Reply:
i added the settings you previously had for both mappings